CODEOWNERS

```diff
@@ -20,6 +23,8 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/network/thrift_proxy @zuercher @brian-pane
 # compressor used by http compression filters
 /*/extensions/filters/http/common/compressor @gsagula @rojkov @dio
+/*/extensions/filters/http/compressor @rojkov @dio
+/*/extensions/filters/http/compressor/gzip @gsagula @rojkov @dio
```

Comment:
Can this and line 25 be removed?

Reply:
Removed the line 27, but 25 is still needed according to `check_format.py fix`